enumerable.rb

```diff
@@ -0,0 +1,110 @@
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable Metrics/PerceivedComplexity
+
+module Enumerable
+  def my_each
+    return to_enum unless block_given
+
+    arr.length.times do |n|
+      yield arr[n]
+    end
+  end
+
+  def my_each_with_index
+    return to_enum unless block_given
+
+    arr.length.times do |n, i|
+      yield(arr[i], n)
+    end
+  end
```

Comment:
- [ ] enumerable #my_each_with_index when a block is given when self is a range returns the range itself after calling the given block once for each element in self

Here is the error I am getting
<img width="674" alt="2" src="https://user-images.githubusercontent.com/58504655/103562715-58d49d80-4edd-11eb-9875-6c20c3ef15d0.PNG">